compiler/hie-core/install.bat

```diff
@@ -0,0 +1,3 @@
+@REM Install hie-core where cabal install would put it on Windows
```

Comment:
You need to add a copyright header here. Running `./fmt.sh` should do that automatically.